packages/components/src/components/multi-select/multi-select/multi-select-utils.ts

```diff
@@ -0,0 +1,205 @@
+import type { FormState } from '../../../utils/form/form-state';
+import type { SelectDropdownDirection, SelectDropdownDirectionInternal } from '../../../utils/select/select-dropdown';
+import { determineDropdownDirection } from '../../../utils/select/select-dropdown';
+import type { Theme } from '../../../utils';
+import { consoleWarn, removeAttribute, setAttribute, setAttributes } from '../../../utils';
+import type { MultiSelectOptionInternalHTMLProps } from '../multi-select-option/multi-select-option-utils';
+import { forceUpdate } from '@stencil/core';
+
+export type MultiSelectState = FormState;
+export type MultiSelectDropdownDirection = SelectDropdownDirection;
+export type MultiSelectOption = HTMLPMultiSelectOptionElement & MultiSelectOptionInternalHTMLProps;
+
+export type MultiSelectUpdateEvent = {
+  value: (string | number)[];
+  name: string;
+};
+
+export const syncMultiSelectOptionProps = (options: MultiSelectOption[], theme: Theme): void => {
+  options
+    .filter((option) => option.theme !== theme)
+    .forEach((option) => {
+      option.theme = theme;
+      forceUpdate(option);
+    });
+};
+
+export const initNativeSelect = (
+  host: HTMLElement,
+  name: string,
+  disabled: boolean,
+  required: boolean
+): HTMLSelectElement => {
+  const nativeSelect = document.createElement('select');
+  setAttributes(nativeSelect, {
+    multiple: 'true',
+    'aria-hidden': 'true',
+    tabindex: '-1',
+    slot: 'select',
+  });
+  syncNativeSelect(nativeSelect, name, disabled, required);
+  host.prepend(nativeSelect);
+  return nativeSelect;
+};
+
+export const syncNativeSelect = (
+  nativeSelect: HTMLSelectElement,
+  name: string,
+  disabled: boolean,
+  required: boolean
+): void => {
+  setAttribute(nativeSelect, 'name', name);
+  // eslint-disable-next-line no-unused-expressions
```

Comment:
I guess that rule is there for a reason ;)
Perhaps something like this works instead:
```
(disabled ? setAttribute ? : removeAttribute)(nativeSelect, 'disabled')
```